modules/monitoring-configurable-monitoring-components.adoc

```diff
@@ -10,7 +10,7 @@ This table shows the monitoring components you can configure and the keys used t
 .Configurable monitoring components
 [options="header"]
 |====
-|Component |cluster-monitoring-config ConfigMap key |user-workload-monitoring-config config map key
+|Component |cluster-monitoring-config ConfigMap key |user-workload-monitoring-config ConfigMap key
```

Comment:
This has twists and turns because we recently updated our [OCP docs guidance](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references) to use `PascalCase` only when referring to the actual object. IIUC in this case, we are referring to a key used within the `ConfigMap` object. 

In other words, the generic ref with lowercase is more appropriate: "config map key". 

I would instead revert the change in this line and `s/ConfigMap key/config map key/` in the `cluster-monitoring-config` row. Does that make sense?